code/modules/antagonists/heretic/knowledge/rust_lore.dm

```diff
@@ -287,12 +287,13 @@
 		return
 
 	var/need_mob_update = FALSE
-	need_mob_update += source.adjustBruteLoss(-5, updating_health = FALSE)
-	need_mob_update += source.adjustFireLoss(-5, updating_health = FALSE)
-	need_mob_update += source.adjustToxLoss(-5, updating_health = FALSE, forced = TRUE)
-	need_mob_update += source.adjustOxyLoss(-5, updating_health = FALSE)
-	need_mob_update += source.adjustStaminaLoss(-20, updating_stamina = FALSE)
+	var/base_heal_amt = 5 * DELTA_WORLD_TIME(SSmobs)
```

Comment:
SSmobs wait is 2 seconds, this doubles the healing received - should be 2.5 multiplied by delta. Also, you've got ``seconds_per_tick`` right there, don't use the define. Same with the other one.

Reply:
`seconds_per_tick` doesn't account for skipped/delayed fires:
```
var/seconds_per_tick = wait / (1 SECONDS)
```
vs
```
#define DELTA_WORLD_TIME(ss) ((world.time - ss.last_fire) * 0.1)
```